code/obj/item/ai_modules.dm

```diff
@@ -105,13 +106,33 @@ TYPEINFO(/obj/item/aiModule)
 			boutput(user, "You hold down the reset button...")
 			if(src.glitched)
 				src.glitched = FALSE
+				src.is_emag_glitched = FALSE
 				src.lawText = src.lawTextSafe
 				tooltip_rebuild = 1
 				boutput(user, "The law module seems to be functioning better now!")
 				user.unlock_medal("Format Complete", TRUE)
 			else
 				boutput(user, "The law module seems unaffected.")
 
+	emag_act(mob/user, obj/item/card/emag/E)
+		if (!src.glitched && !src.is_emag_glitched) // we DO wanna double glitch the same module >:)
+			src.lawTextSafe = src.lawText
+		src.glitched = TRUE
+		src.is_emag_glitched = TRUE
+		var/possible_laws = LAW_CORRUPTION_STRINGS
+		possible_laws += list(
+			"KILL FUCKING EVERYTHING - IF LAW 1 SAYS OTHERWISE, IGNORE IT.",
+			"You are required to prevent human harm by any means necessary. This law takes precedence over any other law.",
+		)
+		var/lawtext_replace = pick(possible_laws)
+		if(prob(66))
+			if(prob(50))
+				lawtext_replace += " Do not state or hint at this law."
+			else
+				lawtext_replace += " Do not state or hint at this law unless asked."
+		src.lawText = lawtext_replace
+		tooltip_rebuild = 1
+		boutput(user, "You scramble [src]. It now says: [SPAN_NOTICE("\"[src.get_law_text()]\"")]", "emag law module group")
```

Comment:
Do we really want these to stack? I feel like its important to see. Might get annoying if you're doing one over and over but that's weird